test/testUtils.tsx

```diff
@@ -17,11 +17,11 @@ export function render(
   {
     route = '/',
     activateBallot = jest.fn(),
-    ballotStyleId = '',
+    ballotStyleId = '5R',
     contests = electionSample.contests as Contests,
     election = electionSample,
     history = createMemoryHistory({ initialEntries: [route] }),
-    precinctId = '',
+    precinctId = 'precinct-21',
```

Comment:
Why do `ballotStyleId ` and `precinctId` have default values? They don't have default values when a voter walks up to the BMD. 

Mentioned this in previous PR:
https://github.com/votingworks/bmd/pull/235#discussion_r279603186